CorsixTH/Lua/entities/machine.lua

```diff
@@ -93,91 +96,130 @@ local function setSmoke(self, isSmoking)
   end
 end
 
+--! Call on machine earthquake impact.
+--!param room (object) machine room
+function Machine:earthquakeImpact(room)
+  self:machineUsed(room)
+end
+
 --! Call on machine use. Handles crashing the machine & queueing repairs
+--!param room (object) machine room
+--!return (bool) is room exploding after this last use
```

Comment:
```suggestion
--!return (bool) is room exploding after this use
```
It is only the `last` use if the room is indeed exploding.

Reply:
> It is only the `last` use if the room is indeed exploding.

Ok